harness/determined/lightning/experimental.py

```diff
@@ -1,4 +1,4 @@
-# type: ignore
+#  type: ignore
```

Comment:
is this intentional?

Reply:
No. I tried to remove it at first, but when I put it back in I mistyped. :|